gameoflife/src/components/LifeCanvas.js

```diff
@@ -25,19 +25,22 @@ class LifeCanvas extends Component {
     let canvas = this.refs.canvas;
     let context = canvas.getContext("2d");
     let boxSize = 20;
+    let fix = canvas.getBoundingClientRect();
 
-    context.fillStyle = "black";
+    context.fillStyle = "grey";
 
     context.fillRect(
-      Math.floor(e.clientX / boxSize) * boxSize,
-      Math.floor(e.clientY / boxSize) * boxSize,
+      Math.floor((e.clientX - fix.x) / boxSize) * boxSize,
```

Comment:
Looking good here, nice that you found a way to offset it by position on the page!